inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_check.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gna/gna_config.hpp>
+
+class GnaLayerTestCheck : virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
maybe this class could be a template class ?

Reply:
Please see comment above.